src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[] {
+                new Truck(),
+                new Bulldozer(),
+                new Excavator()
+        };
+        for (Machine part : machines) {
+            part.doWork();
+        }
+        for (Machine part : machines) {
+            part.stopWork();
```

Comment:
Similarly, the messages inside the 'stopWork' method should be informative and indicate which type of Machine has stopped working.